ThePensionsRegulator.Frontend/Styles/_tpr-button.scss

```diff
@@ -38,106 +36,110 @@ govuk-button:focus:not(:active):not(:hover) {
     padding-left: $tpr-button-padding;
     padding-right: $tpr-button-padding;
 }
+
 govuk-button:focus:not(:active):not(:hover) {
     color: $tpr-colour-eclipse;
 }
 
-    .govuk-button:hover, .govuk-button:active {
-        background-color: $tpr-colour-violet;
-    }
+.govuk-button:hover, .govuk-button:active {
+    background-color: $tpr-colour-violet;
+}
 
-    .govuk-button:disabled, .govuk-button[aria-disabled=true] {
-        background-color: $tpr-colour-regent-st-blue;
-    }
+.govuk-button:disabled, .govuk-button[aria-disabled=true] {
+    background-color: $tpr-colour-regent-st-blue;
+}
 
-    .govuk-button--secondary, .govuk-file-upload::file-selector-button {
-        color: $tpr-colour-royal-blue;
-        background: $tpr-colour-white;
-        border-color: $tpr-colour-royal-blue;
-        box-shadow: none;
-    }
-    // Only for links styled as secondary buttons, we need the extra weight of the a. selector to override GOV.UK color
-    a.govuk-button--secondary:link, a.govuk-button--secondary:visited {
-        color: $tpr-colour-royal-blue;
-    }
+.govuk-button--secondary, .govuk-file-upload::file-selector-button {
+    color: $tpr-colour-royal-blue;
+    background: $tpr-colour-white;
+    border-color: $tpr-colour-royal-blue;
+    box-shadow: none;
+}
+// Only for links styled as secondary buttons, we need the extra weight of the a. selector to override GOV.UK color
+a.govuk-button--secondary:link, a.govuk-button--secondary:visited {
+    color: $tpr-colour-royal-blue;
+}
 
-    .govuk-button--secondary:hover,
-    .govuk-button--secondary:link:active,
-    .govuk-button--secondary:visited:active,
-    a.govuk-button--secondary:hover {
-        color: $govuk-button-text-colour;
-        background-color: $tpr-colour-royal-blue;
-    }
+.govuk-button--secondary:hover,
+.govuk-button--secondary:link:active,
+.govuk-button--secondary:visited:active,
+a.govuk-button--secondary:hover {
+    color: $govuk-button-text-colour;
+    background-color: $tpr-colour-royal-blue;
+}
 
-    .govuk-button--secondary:disabled, .govuk-button--secondary[aria-disabled=true] {
-        color: $tpr-colour-cornflower;
-        background-color: $tpr-colour-white;
-        border-color: $tpr-colour-cornflower;
-    }
-    // Remove chevron where it's not wanted
-    .govuk-button--start,
-    .govuk-button--secondary {
-        background-image: none;
-        padding-right: $tpr-button-padding;
-    }
+.govuk-button--secondary:disabled, .govuk-button--secondary[aria-disabled=true] {
+    color: $tpr-colour-cornflower;
+    background-color: $tpr-colour-white;
+    border-color: $tpr-colour-cornflower;
+}
+// Remove chevron where it's not wanted
```

Comment:
Looks like this rule can be removed. The start button should still have a chevron - I guess this is removing a second chevron.

Reply:
The following group of selectors (was Lines 83-87, now 74-79) are also removing the chevron where it's not wanted.